experiment/compatibility-versions/e2e-k8s-compatibility-versions.sh

```diff
@@ -243,6 +241,25 @@ EOF
     --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/-", "value": "--v='"${KIND_CLUSTER_LOG_LEVEL}"'" }]'
 }
 
+build_tests() {
+  echo "Building e2e.test binary from release branch ${PREV_RELEASE_BRANCH}..."
+  # Ensure the e2e test binary is built from the *current* (previous release) checkout
+  if ! make all WHAT="test/e2e/e2e.test vendor/github.com/onsi/ginkgo/v2/ginkgo"; then
+      echo "WARN: 'make all WHAT=...' failed, attempting older 'make WHAT=test/e2e/e2e.test'..."
```

Comment:
was this failing? what version? make all WHAT= is supposed to support multiple targets for a long time now

Reply:
+1, I hit an error but it was due to accidentally inputting a much old version in the test env var.  This isn't necessary given compatibility version rolling n-<1...3>.  I've removed this now.